src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -588,7 +588,7 @@ static void window_ride_construction_close(rct_window* w)
     gMapSelectFlags &= ~MAP_SELECT_FLAG_ENABLE_ARROW;
 
     // In order to cancel the yellow arrow correctly the
-    // selection tool should be cancelled. Don't do a tool cancel if
+    // selection tool should be canceled. Don't do a tool cancel if
```

Comment:
cancelled is correct in ~~proper~~British English.

Reply:
Damn, that slipped it just annoyed me as VS kept outlining it.